lib/modules/manager/gitlabci/extract.spec.ts

```diff
@@ -57,7 +62,7 @@ describe('modules/manager/gitlabci/extract', () => {
       ]);
       expect(res).toMatchSnapshot();
       expect(res).toHaveLength(1);
-      expect(res[0].deps).toHaveLength(4);
+      expect(res[0].deps).toHaveLength(10);
```

Comment:
Were we missing a lot before? I.e. previous implementation was incomplete/buggy?

Reply:
no, there are 6 new services in fixture file.